mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/cfg/spi/StandalonePojoMapperSpiSettings.java

```diff
@@ -18,12 +18,17 @@ private StandalonePojoMapperSpiSettings() {
 
 	public static final String BEAN_PROVIDER = PREFIX + Radicals.BEAN_PROVIDER;
 
+	public static final String INTEGRATION_PARTIAL_BUILD_STATE =
+			PREFIX + Radicals.INTEGRATION_PARTIAL_BUILD_STATE;
+
 	public static class Radicals {
 
 		private Radicals() {
 		}
 
 		public static final String BEAN_PROVIDER = "bean_provider";
+
+		public static final String INTEGRATION_PARTIAL_BUILD_STATE = "integration_partial_build_state";
```

Comment:
This wouldn't be a problem if one day we decide to mix ORM and standalone mappers? 🙈

Reply:
It might, yes, but... we're still a long way from that :)
Also, I exposed this as SPI to mirror what we do the the ORM mapper, but if you look at the implementation, technically this property is internal: it's only ever set or consumed by us, not by users or service providers. So I guess we could change its name fairly easily.